SecurityManager/Clients.Web.Test.SetupDB/SetupDB.sql

```diff
@@ -165,12 +165,12 @@ CREATE TYPE [dbo].[TVP_Double_Distinct] AS TABLE
   UNIQUE CLUSTERED ([Value])
 )
 GO
-CREATE TYPE [dbo].[TVP_DateTime] AS TABLE
+CREATE TYPE [dbo].[TVP_DateTime2] AS TABLE
 (
   [Value] datetime NULL
 )
 GO
-CREATE TYPE [dbo].[TVP_DateTime_Distinct] AS TABLE
+CREATE TYPE [dbo].[TVP_DateTime2_Distinct] AS TABLE
 (
   [Value] datetime NULL
```

Comment:
Is it intentional that we did not change the column types to `datetime2`?

Reply:
Noep